ml_git/relationship/entity_manager.py

```diff
@@ -137,3 +137,29 @@ def get_entity_versions(self, entity_name, metadata_repo_name):
             entity_version = SpecVersion(spec_tag_yaml)
             versions.append(entity_version)
         return versions
+
+    def get_linked_entities(self, entity_name, entity_version, metadata_repo_name):
+        """Get a list of linked entities found for an entity version.
+
+        Args:
+            entity_name (str): The name of the entity you want to get the linked entities.
+            entity_version (str): The version of the entity you want to get the linked entities.
```

Comment:
```suggestion
            name (str): The name of the entity you want to get the linked entities.
            version (str): The version of the entity you want to get the linked entities.
```
In other parts of this code we use the variable name without the entity. Unless it is not valid for this scenario, I think we can keep the same pattern.

Reply:
Yes, done on commit: https://github.com/HPInc/ml-git/pull/115/commits/d6f30e8e5d15e7e96d06733c9182e0872c051978